src/walktree.js

```diff
@@ -1,49 +1,60 @@
-const fs = require('fs');
+var fs = require('fs');
+var path = require('path');
 
-const walkTree = function walkTree(opts) {
-    const path = opts.root;
-    const ignored = opts.ignore;
-    const create = opts.create;
-    const countFolders = opts.countFoldersAsFiles;
-    const fileName = opts.file;
 
-    walkTreeRecursive(path);
+function walkTree (opts) {
+    var create = opts.create;
+    var ignored = opts.ignore;
+    var countFolders = opts.countFoldersAsFiles;
+    var nameOfPlaceholder = opts.file;
 
-    function walkTreeRecursive(path) {
-        let elements = fs.readdirSync(path);
-        let fileCount = 0;
+    walkTreeRecursive(opts.root);
```

Comment:
Before, you had used the variable name `path` here, as well as re-declaring it as an argument inside your `walkTreeRecursive` function. Even when using scoped variables, this is generally bad practice. Here, since Node will redeclare your variable with whatever value was used inside the loop, the value is no longer what you expected  once you're out of your loop